quackfactory/cogs/session.py

```diff
@@ -21,7 +22,7 @@ async def on_voice_state_update(member, before, after):
         for member in guild.members:
             for role in member.roles:
                 if role.name == "Test": # Rubberduck role idk why but it doesnt work using a variable 

```

Comment:
Perhaps you could compare the [role id](https://discordpy.readthedocs.io/en/latest/api.html?highlight=role%20id#discord.Role.id) to the named variable for the role.

Reply:
Alright.